images/wordpress/scripts/entrypoint.sh

```diff
@@ -18,6 +18,8 @@ add-preset-data
 
 reset-permissions
 
+start-nginx
```

Comment:
kun je dit doen op basis van een environment variable ofzo? want nu gaat er wel nginx draaien in onze dev containers